backend/arkham-api/library/Arkham/Location/Cards/RainyLondonStreets.hs

```diff
@@ -0,0 +1,35 @@
+module Arkham.Location.Cards.RainyLondonStreets (rainyLondonStreets, RainyLondonStreets(..)) where
```

Comment:
```suggestion
module Arkham.Location.Cards.RainyLondonStreets (rainyLondonStreets) where
```

One thing newer code tries to avoid is exporting the constructor unless we make a new card that enhances it (like ReturnTos or when a card is printed again, like Noxious Fumes in the midwinter gala).